apps/freeablo/farender/renderer.cpp

```diff
@@ -277,9 +277,8 @@ namespace FARender
                                   mItems,
                                   state->mPos.current().x,
                                   state->mPos.current().y,
-                                  state->mPos.next().x,
-                                  state->mPos.next().y,
-                                  state->mPos.getDist());
+                                  state->mPos.getFractionalPos().x,
+                                  state->mPos.getFractionalPos().y);

```

Comment:
This would be cleaner (and simpler math in places) if `Misc::Point` stored `FixedPoint` coordinates rather than `int32_t`, i.e. currently the calculations have to be done in two parts (int and fractional), and two parameters passed (int and fractional) where it should be just one